web/src/user_profile.js

```diff
@@ -859,6 +864,11 @@ export function show_edit_user_info_modal(user_id, $container) {
                 hide_button_spinner($submit_btn);
                 $submit_btn.prop("disabled", true);
                 original_values = get_current_values($("#edit-user-form"));
+                ui_report.success(
+                    $t_html({defaultMessage: "Saved successfully!"}),
+                    $("#user-profile-modal .save-success"),
+                    1200,
+                );
```

Comment:
`settings_components.change_save_button_state` uses `Saved` as the text; I don't see a great reason to diverge from that...

Reply:
Fixed it.